examples/player/simid_player.js

```diff
@@ -284,6 +283,21 @@ class SimidPlayer {
     return newDimensions;
   }
 
+  /** Validates and displays the non-linear creative. */
+  displayNonlinearCreative_() {
+    const newDimensions = this.getCreativeDimensions_();
+    
+    if (!this.isValidDimensions_(newDimensions)) {
+      console.log('Unable to play a non-linear ad with dimensions bigger than the player. Please modify dimensions to a smaller size.');
```

Comment:
there is a sendLog function in this file, perhaps utilize it here

Reply:
I think sendLog might not work here. Because sendLog sends the message to the creative, but this might not be relevant to the creative.

If I'm not mistaken this message is related to the setup of the player only, and it's possible that the creative doesn't even exist yet.

Reply:
nothing is initialized at this point so can't use that